Druid/DRSCCP.class.st

```diff
@@ -200,9 +200,9 @@ DRSCCP >> latticeValue: aValue [
 DRSCCP >> latticeValueOfPhiFunction: operation [
 
 	| values |
-
-	values := operation operands collect: [ :e | 
-		self latticeValue: e ] as: Set.
+	values := (operation operands
+		           reject: [ :e | e isNullValue ] "Avoid NULLs"
```

Comment:
Can we have nulls???

Reply:
Yes, I think it comes from messages like `ifNotNil`

Reply:
ah yes